### PR TITLE
pass e2e 

### PR DESCRIPTION
**Pull request by gadcl**
setup fix - config for matic reader + ganache increase warm up

### CODE REVIEW

**Comment by uv-orbs** [comment]
hey gad
this line repeats twice in render nodes
maticConfig.EthereumGenesisContract = '0x35eA0D75b2a3aB06393749B4651DfAD1Ffd49A77';
145+147
can u pls update?

File: src/api/render-node.ts [hunk 1]
```diff
@@ -137,6 +137,20 @@ function getMaticReader(snapshot: StateSnapshot, config: ServiceConfiguration) {
...
+    ...config.ExternalLaunchConfig,
+    BootstrapMode: false,
+    };
+    maticConfig.Port =8080;
+    maticConfig.EthereumGenesisContract = '0x35eA0D75b2a3aB06393749B4651DfAD1Ffd49A77';
+    maticConfig.EthereumEndpoint = config.MaticEndpoint ?? 'https://matic-router.global.ssl.fastly.net';
+    maticConfig.EthereumGenesisContract = '0x35eA0D75b2a3aB06393749B4651DfAD1Ffd49A77';
+    maticConfig.EthereumFirstBlock = 21700000;
+    maticConfig['node-address'] =  config['node-address'];
+    maticConfig.DeploymentDescriptorPollIntervalSeconds = 600;
```
Comment (amihaz) [inline src/api/render-node.ts:150]: I think it's better to change the param in test

File: src/api/render-node.ts [hunk 2]
```diff
@@ -137,6 +137,20 @@ function getMaticReader(snapshot: StateSnapshot, config: ServiceConfiguration) {
...
+    maticConfig.EthereumPollIntervalSeconds = 10;
```
Comment (amihaz) [inline src/api/render-node.ts:151]: also here maybe better change it in test?

File: src/api/render-node.ts [hunk 3]
```diff
@@ -137,6 +137,20 @@ function getMaticReader(snapshot: StateSnapshot, config: ServiceConfiguration) {
   const imageTag = parseImageTag(version);
   if (!imageTag) return undefined;
 
+  const maticConfig: {[index: string]:any} = {
+    ...config.ExternalLaunchConfig,
+    BootstrapMode: false,
```
Comment (amihaz) [inline src/api/render-node.ts:142]: I think this is a bug which will cause management-service to wakeup in bootstrap mode

**Pull request merged**